lib/fhir_models/fhir_ext/structure_definition.rb

```diff
@@ -559,7 +554,7 @@ def is_data_type?(data_type_code, value)
       when 'domainresource'
         true # we don't have to verify domain resource, because it will be included in the snapshot
       when 'boolean'
-        value == true || value == false || value.downcase == 'true' || value.downcase == 'false'
+        value == true || value == false || value.casecmp('true').zero? || value.casecmp('false').zero?
```

Comment:
I'm not a huge fan of this change from a readability perspective. To me, it's more obvious what the goal is when we downcase a variable and compare it to `'true'` than it is when you run `casecmp` and check if it's zero. Is there a performance benefit to using `casecmp`?